### ISSUE TITLE
ChoiceChips 

### ISSUE THREAD

**Issue by muhammadibcode** [FlutterFlow/flutterflow-issues#3917]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have ChoiceChips in my widget tree with multi select on, but I am unable to select it as a widget in a Clear All/Select All action on the same page.

### Expected Behavior

I expected to be able to select the ChoiceChips widget and clear the user's current selection.

### Steps to Reproduce

1. Create new page
2. Add ChoiceChips widget with multi choice enabled to ListView
3. Add Clear All/Select All action
4. Try to select ChoiceChips widget

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFXhfHlvIV2oeZE0Zf9LMZWgmQwGVgLaJJEktYbdzwYCLbpPe9/dsjRNxdWTe27YGlhHFabvGYywa3vkvzlLvYoPTObfJhY+sxiTwmVWXq8Rq6tPpbtQHJRBfNUIEyY4cGZnR1/Ns9pWGQgwFyIJfKtfCvZN5+/Zwh9f6PHaOI=

### Visual documentation

<img width="1088" alt="Screenshot 2024-09-27 at 18 06 30" src="https://github.com/user-attachments/assets/9b7e2ebd-9aa3-4c9f-9b33-5bf78e72abe3">
<img width="1120" alt="Screenshot 2024-09-27 at 18 07 17" src="https://github.com/user-attachments/assets/33e53fde-bc07-474b-ae5b-4b01c5bb5434">
<img width="1022" alt="Screenshot 2024-09-27 at 18 07 57" src="https://github.com/user-attachments/assets/7afda934-0cbf-4e06-87f4-84e5a5216ce0">

### Environment

```markdown
- FlutterFlow version: 5.0.1
- Platform: MacOS Desktop
- Browser name and version: Not using a browser
- Operating system and version affected: MacOS Sonomo 14.5
```

### Additional Information

Blocking from fixing critical bug in my app

**Comment by muhammadibcode** [comment]
Adding that the issue seems to be when ChoiceChips are shown in a ListView

**Comment by rzambroni** [comment]
Hi there,

Thank you for bringing this issue to our attention! I attempted to replicate the problem in a blank project, including with the chips inside a ListView, but I was unable to reproduce the behavior.

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/d210c22c-0175-4470-b88b-3dc7f5fba6ab/dddfb47b-45de-47fd-830e-bf76cc4f27ca?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9kMjEwYzIyYy0wMTc1LTQ0NzAtYjg4Yi0zZGM3ZjVmYmE2YWIvZGRkZmI0N2ItNDVkZS00N2ZkLTgzMGUtYmY3NmNjNGYyN2NhIiwiaWF0IjoxNzI3NDY2OTM0LCJleHAiOjMzMjk4MDI2OTM0fQ.VuKt9hmQaoZt3Fh2mtbT3nKW_0HNPfrWAM-_k_-91S0 " alt="image.png" width="1300" height="978" />

Additionally, after reviewing your project, I noticed there are errors related to the ChoiceChips. I recommend addressing these issues first, as they might be impacting the ability to select the chips in an action.

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/b1ed41d0-c5e5-4471-82e1-84e96444f25a/f43d1a3f-ab3c-41b3-b49a-2a99aaa118a9?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9iMWVkNDFkMC1jNWU1LTQ0NzEtODJlMS04NGU5NjQ0NGYyNWEvZjQzZDFhM2YtYWIzYy00MWIzLWI0OWEtMmE5OWFhYTExOGE5IiwiaWF0IjoxNzI3NDY2OTM0LCJleHAiOjMzMjk4MDI2OTM0fQ.03rbCKv3M6hisxUhS2xLRtZlRfpw9gV-wFMsMsfdCgs " alt="image.png" width="319" height="379" />
